datacommons/statvar.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Data Commons Python API Stat Module.
+
+Provides functions for getting data on StatVars from Data Commons Graph.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from datacommons.utils import _API_ROOT, _API_ENDPOINTS, _ENV_VAR_API_KEY
+
+import json
+import os
+import six.moves.urllib.error
+import six.moves.urllib.request
+
+import datacommons.utils as utils
+
+
+def _send_get_stat_req(url):
+
+    headers = {'Content-Type': 'application/json'}
+    if os.environ.get(_ENV_VAR_API_KEY):
+        headers['x-api-key'] = os.environ[_ENV_VAR_API_KEY]
+
+    req = six.moves.urllib.request.Request(url, headers=headers)
+
+    try:
+        res = six.moves.urllib.request.urlopen(req)
+    except six.moves.urllib.error.HTTPError as e:
+        raise ValueError('Response error {}:\n{}'.format(e.code, e.read()))
+
+    # Verify then store the results.
+    res_json = json.loads(res.read())
+    return res_json
+
+
+def get_stat_value(place,
+                   stat_var,
+                   date=None,
+                   measurement_method=None,
+                   observation_period=None,
+                   unit=None,
+                   scaling_factor=None):
+    """Returns a value for :code:`place` based on the :code:`stat_var`.
+
+    Args:
+      place (:obj:`iterable` of :obj:`str`): The dcid of `Place` to query for.
+      stat_var (:obj:`str`): The dcid of the `StatisticalVariable`.
+      date (:obj:`str`): Optional, the preferred date of observation
+        in ISO 8601 format. If not specified, returns the latest observation.
+      measurement_method (:obj:`str`): Optional, the dcid of the preferred
+        `measurementMethod` value.
+      observation_period (:obj:`str`): Optional, the preferred
+        `observationPeriod` value.
+      unit (:obj:`str`): Optional, the dcid of the preferred `unit` value.
+      scaling_factor (:obj:`int`): Optional, the preferred `scalingFactor` value.
+    Returns:
+      A :obj:`double` the value of :code:`stat_var` for :code:`place`, filtered
```

Comment:
in python it should be "float" instead of double

Reply:
Thanks, changed